superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -55,6 +55,7 @@ import DashboardContainer from './DashboardContainer';
 
 const TABS_HEIGHT = 47;
 const HEADER_HEIGHT = 67;
+const NATIVE_FILTER_BAR_WIDTH = 255;
```

Comment:
Why 255 instead of 250?

Reply:
To fix the inner margin-right of the filter values. They have a fixed width.